test/TestCases/Expressions/Add.cpp

```diff
@@ -48,12 +48,15 @@ TEMPLATE_PRODUCT_TEST_CASE ("Addition in place", "[VCTR][add]", (PlatformVectorO
 
     auto sum1 = srcA;
     auto sum2 = srcA;
+    auto sum3 = srcA;
 
     sum1 += srcB;
     sum2 += c;
+    sum3 = vctr::addConstant<ElementType (42)> << srcA;
```

Comment:
You put this in the "Addition in place" section which seems to not really fit to me ;)